server/offenceCodeDecisions/Decision.ts

```diff
@@ -86,15 +86,16 @@ export class Decision {
     return this.findDecisionBy(d => d.getUrl() === url)
   }
 
+  findDecisionById(id: string): Decision {
```

Comment:
This is function just for tests - consider moving

Reply:
Yeah - I'll pull that out